src/app/onion/dashboard/dashboard.component.ts

```diff
@@ -694,7 +694,6 @@ export class DashboardComponent implements OnInit, OnDestroy {
   async addToCollection(collection?: string) {
     if (collection) {
       this.collectionService.submit(this.focusedLearningObject.id, collection).then(() => {
-        this.focusedLearningObject.publish();
```

Comment:
Can we verify that we don't have any template logic that uses this?

Reply:
from what I've seen from looking through and testing the dashboard after creating a learning object it doesn't look like there's any template logic there but @nrwinner might have more context.

Reply:
Did this get resolved?

Reply:
It seems to be in order, but we might want to get @nrwinner to stage this on his machine and give it a run through to be sure.

Reply:
I agree